### PR TITLE
DEVDOCS-4649: [update] Stencil CLI logging options

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4649]

## What changed?
added --no-verbose and --verbose to stencil start

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4649]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4649?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ stencil init [--url STORE_URL] [--token API_TOKEN]
 Starts the live theme preview using the theme files in the current directory.
 
 ```shell title="Usage: stencil start"
-stencil start [-V|--version] [-o|--open] [-v|--variation] [-t|--test] [-t|--tunnel]
+stencil start [-V|--version] [-o|--open] [-v|--variation] [-t|--test] [-t|--tunnel] [-nov|--no-verbose] [-vb|--verbose]
```
Comment (bc-max) [inline docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md:70]:
```suggestion
stencil start [-V|--version] [-o|--open] [-v|--variation] [-t|--test] [-t|--tunnel] [-vb|--verbose]
```
We will provide `no-verbose` mode by default, so there is no need for this option any more.

File: docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md [hunk 2]
```diff
@@ -77,12 +77,14 @@ stencil start --open # opens live theme preview in default browser
...
+| `--version`                  |`-V` | Output the version number                                                             |
 | `--open`                     |`-o` | Automatically open default browser                                                    |
-| `--variation NAME`       |`-v` | Set which theme variation to use while developing                                     |
+| `--variation NAME`           |`-v` | Set which theme variation to use while developing                                     |
 | `--test`                     |`-t` | Enable QA mode which will bundle all javascript for speed to test locally             |
 | `--tunnel`                   |     | Create a tunnel URL which points to your local server which anyone can use            |
-| `--no-cache`                 |`-n` | Turn off caching for API resource data (cache refreshes every 5 minutes)             |
+| `--no-cache`                 |`-n` | Turn off caching for API resource data (cache refreshes every 5 minutes)              |
+| `--no-verbose`               | `-nov`| Disable verbose logging                                                             |
```
Comment (bc-max) [inline docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md:86]: We can remove line 86 completely

**Comment by bc-max** [comment]
@bc-traciporter Taking into account we introduce a breaking change and will release our code under new major version, we also need to merge devdocs ticket more or less simultaneously with `Stencil-cli` release. Do you have any procedure for such a case or how this usually works? Your thoughts are highly appreciated.

File: docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md [hunk 3]
```diff
@@ -77,12 +77,13 @@ stencil start --open # opens live theme preview in default browser
...
 |-|-|-|
-| `--version`                  |`-V` | Output the version number                                                            |
+| `--version`                  |`-V` | Output the version number                                                             |
 | `--open`                     |`-o` | Automatically open default browser                                                    |
-| `--variation NAME`       |`-v` | Set which theme variation to use while developing                                     |
+| `--variation NAME`           |`-v` | Set which theme variation to use while developing                                     |
 | `--test`                     |`-t` | Enable QA mode which will bundle all javascript for speed to test locally             |
 | `--tunnel`                   |     | Create a tunnel URL which points to your local server which anyone can use            |
-| `--no-cache`                 |`-n` | Turn off caching for API resource data (cache refreshes every 5 minutes)             |
+| `--no-cache`                 |`-n` | Turn off caching for API resource data (cache refreshes every 5 minutes               |
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md:85]:
```suggestion
| `--no-cache`                 |`-n` | Turn off caching for API resource data (cache refreshes every 5 minutes)              |
```

**Comment by slsriehl** [comment]
@bc-traciporter one suggestion from me & then I'm gtg

**Review by slsriehl — DISMISSED** [review]
ready for product launch

**Review by slsriehl — APPROVED** [review]

**Pull request merged**